src/components/tile/tile.style.js

```diff
@@ -0,0 +1,86 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../style/themes/base';
+
+const paddingSizes = {
```

Comment:
Could you move it to the Options-Helper please? :) And then we can import it also in tests (https://github.com/Sage/carbon/pull/2260/files#diff-453b70e8319831a4c9f04c6bed98ac5fR168)

Reply:
The values that the prop can take (`XS`, `S`, `M`, `L`, `XL`) are in the optionsHelper, via `OptionsHelper.sizesTile`. But I don't think the actual resulting padding values don't need to be stored in there as the optionsHelper is just for the values the prop takes?

And I'm using them in a slightly different format in the specs, so I couldn't just flat import the same thing to use in both.